package.json

```diff
@@ -49,20 +49,20 @@
     "@angular/platform-browser": "7.2.15",
     "@angular/platform-browser-dynamic": "7.2.15",
     "@angular/router": "7.2.15",
-    "@blackbaud/auth-client": "2.17.0",
+    "@blackbaud/auth-client": "2.17.1",
     "@pact-foundation/pact": "7.4.0",
     "@pact-foundation/pact-web": "7.4.0",
-    "@skyux-sdk/builder": "3.7.1",
+    "@skyux-sdk/builder": "3.11.0",
     "@skyux-sdk/builder-plugin-skyux": "1.0.0",
     "@skyux-sdk/e2e": "3.1.2",
     "@skyux-sdk/pact": "3.2.1",
     "@skyux-sdk/testing": "3.1.0",
     "@skyux/animations": "3.0.0",
     "@skyux/assets": "3.1.0",
-    "@skyux/config": "3.5.0",
+    "@skyux/config": "3.6.0",
     "@skyux/core": "3.8.0",
     "@skyux/forms": "3.2.1",
-    "@skyux/http": "3.4.1",
+    "@skyux/http": "3.7.0",
```

Comment:
Why bump up all these packages?

Reply:
I believe it was a requirement of bumping up builder which I needed because I had the expired certs problem.

Reply:
Ah, yes. Thanks for doing that 👍